.github/workflows/CI.yml

```diff
@@ -335,15 +336,43 @@ jobs:
         uses: aws-observability/aws-otel-collector-test-framework@deprecating
         with:
           running_type: integ-test
-          opts: "-t=EC2_TEST -s=build/packages/.aoc-stack-test.yml -a=${{ matrix.ami }}"
-
-  e2etest-ecs:
-    strategy:
-      matrix:
-        type: [EC2, FARGATE]
+          opts: "-t=EC2_TEST -s=build/packages/.aoc-stack-test.yml -a=${{ matrix.ami }}" 
+             
+  get-testing-suites:
+    runs-on: ubuntu-latest
+    outputs:
+      eks-matrix: ${{ steps.set-matrix.outputs.eks-matrix }}
+      ecs-matrix: ${{ steps.set-matrix.outputs.ecs-matrix }}
+      ec2-matrix: ${{ steps.set-matrix.outputs.ec2-matrix }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      
+      - name: Setup Python
+        uses: actions/setup-python@v2.1.4
+
+      - name: Get all the testing suites
+        id: set-matrix
+        run: | 
+          testing_suites=$(python -c 'import os, json; print(json.dumps(os.listdir("e2etest/receiver")))')
```

Comment:
should we use a general name for the folder name instead of  'receiver' ? it looks we could test a specific config for processors or exporters in the future

Reply:
yes we should, let me change it